From 3587e8ff1573deefa67ab94fc851656dc42adac2 Mon Sep 17 00:00:00 2001
From: Duy Bao Dang <duy.dang.yb@renesas.com>
Date: Fri, 31 Jul 2020 09:33:27 +0700
Subject: [PATCH 252/433] ARM: shmobile: r8a7743: Add clock index marco for 3DX
 graphics module

Add clock index macro for the 3DX graphics module (PowerVR SGX544MP2) of
the R8A7743 (RZ/G1M) SoC.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
Signed-off-by: Duy Bao Dang <duy.dang.yb@renesas.com>
---
 include/dt-bindings/clock/r8a7743-clock.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dt-bindings/clock/r8a7743-clock.h b/include/dt-bindings/clock/r8a7743-clock.h
index 8e4b843..2db7888 100644
--- a/include/dt-bindings/clock/r8a7743-clock.h
+++ b/include/dt-bindings/clock/r8a7743-clock.h
@@ -22,6 +22,7 @@
 #define R8A7743_CLK_SD0			7
 #define R8A7743_CLK_Z			8
 #define R8A7743_CLK_RCAN		9
+#define R8A7743_CLK_SGX			10
 
 /* MSTP0 */
 #define R8A7743_CLK_MSIOF0		0
-- 
2.7.4

